plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -163,7 +155,20 @@ def sweep(self, arguments: List[str]) -> None:
             if param_name in search_space:
                 del search_space[param_name]
 
-        sampler = self.get_sampler()
+        samplers = {
+            "tpe": "TPESampler",
+            "random": "RandomSampler",
+            "cmaes": "CmaEsSampler",
+        }
+        if self.optuna_config.sampler.name not in samplers:
+            raise NotImplementedError(
+                f"{self.optuna_config.sampler} is not supported by Optuna sweeper."
+            )
+
+        sampler_class = getattr(
+            optuna.samplers, samplers[self.optuna_config.sampler.name]
+        )
```

Comment:
You can use 

try:

```python
from hydra.utils import get_class

sampler_class = get_class("optuna.samplers.TPESampler")
```

Reply:
Thanks! I applied your suggestion in https://github.com/facebookresearch/hydra/pull/1132/commits/fd3403baa6020b49ff3a801a096e0951c98844c9.